### ISSUE TITLE
Remove non-working old notebooks

### ISSUE THREAD

**Issue by benjaminleighton** [rom-py/rompy-legacy#77]
For now can we remove everything except the subdirectories at https://github.com/rom-py/rompy/tree/main/notebooks? @rafa-guedes and @tomdurrant?

**Comment by tomdurrant** [comment]
Yes, I think we can. But arguably we should remove all notebook from this repo, and make a separate examples repo (see #61).

**Comment by rafa-guedes** [comment]
@benjaminleighton I have removed some of the legacy swan ones in https://github.com/rom-py/rompy/pull/66. There is still some cleanup to be done. Agree with @tomdurrant we could move notebooks into a standalone repo in rom-py organisation to avoid this repo becoming too large.
